kata-deploy/Dockerfile

```diff
@@ -1,13 +1,14 @@
 FROM centos/systemd
-ARG KATA_VER=1.1.0
+ARG KATA_VER=1.2.0
 ARG KATA_URL=https://github.com/kata-containers/runtime/releases/download/${KATA_VER}
+ARG KATA_FILE=kata-static-${KATA_VER}-x86_64.tar.xz
```

Comment:
I wonder if we want to put that architecture in a variable (or rename the file to `Dockerfile.x86_64`)?

Reply:
I'll set it to a variable